styles.css

```diff
@@ -71,6 +113,38 @@ body {
   transition: opacity .5s;
 }
 
+.main__button::after {
+  position: absolute;
+  margin: 35% 77% 0 16%;
+  padding-top: 6px;
+  padding-right: 0;
+  width: 200px;
+  height: 22px;
+  border-radius: 4px;
+  border: 3px solid #0056ec;
+  display: block;
+  top: 0;
+  left: 0;
+  content: '';
+animation: pulse1 1s linear infinite;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6357730)